admin_guide/high_availability.adoc

```diff
@@ -10,50 +10,163 @@
 toc::[]
 
 == Overview
-This document describes how to setup high available services on your
+This topic describes how to set up highly-available services on your
 OpenShift cluster.
 
-== IP Failover
-This section describes how to make any IP based network service highly available using virtual IP addresses and IP failover.
+The Kubernetes link:../architecture/core_objects/kubernetes_model.html#replication-controller[replication controller] ensures that the deployment requirements,
+in particular the number of replicas, are satisfied when the appropriate
+resources are available. When run with two or more replicas, the link:../architecture/core_objects/routing.html[router] can be resilient to failures, providing a highly-available service. Depending on how the router instances are discovered (via a service,
```

Comment:
In the second sentence, maybe start it with "For example, when run with two or more replicas...", since the router is just one example of implementing HA, right? Without that context, it might seem like this topic is just going to be all about routers.